demo/AppNavigator.js

```diff
@@ -0,0 +1,35 @@
+import { createStackNavigator, createAppContainer } from "react-navigation";
```

Comment:
@marcelomorgado Did you use `git mv` to do this? It's weird they didn't recognize this as a file renaming?

https://git-scm.com/docs/git-mv

Reply:
Sorry, I've used fs mv cmd.

Reply:
Ah ok. Do you agree that using `git mv` in the future would make sense? Thanks!

Reply:
Sure, absolutely!